vividus-plugin-web-app/src/main/resources/properties/profile/web/desktop/chrome/mobile_emulation/phone/profile.properties

```diff
@@ -1,5 +1,5 @@
 bdd.all-meta-filters=groovy: layout && layout.toString().tokenize(' ').contains('chrome_phone') && !skip && (${bdd.meta-filters})
 
-web.driver.CHROME.experimental-options={"mobileEmulation": {"deviceName": "${org.vividus.selenium.WebDriverFactory-mobileEmulation/deviceName}"}}
-org.vividus.selenium.WebDriverFactory-mobileEmulation/deviceName=iPhone 8
+web.driver.CHROME.experimental-options={"mobileEmulation": {"deviceName": "${org.vividus.selenium.mobileEmulation.deviceName}"}}
+org.vividus.selenium.mobileEmulation.deviceName=iPhone 8
```

Comment:
`web.driver.CHROME.mobile-emulation.device-name` and similar approach for other properties